pkg/common/serviceprovider/azure/provider.go

```diff
@@ -0,0 +1,14 @@
+package azure
```

Comment:
there is a choice that we keep the network side in the Azure org as it is mostly not easy to extend network implementation